src/amo/components/Addon/index.js

```diff
@@ -154,22 +151,18 @@ export class AddonBase extends React.Component {
     this.props.toggleThemePreview(event.currentTarget);
```

Comment:
I think we should remove this as well, no?

Reply:
It looks like we can also remove the entire `componentWillUnmount` method as it looks like that's entirely about dealing with theme previews.